test/simple-filtered-collection-test.js

```diff
@@ -63,7 +66,7 @@ describe('SimpleFilteredCollection', function() {
       var model = new Backbone.Model({ i: 1 })
       baseCollection.add(model);
       assert.strictEqual(filteredCollection.length, 0);
-      events = [];
+      events.length = 0;
```

Comment:
The reason is that since I moved the event setup into a function, I can't "reset" the collection by replacing it, but need to truncate it instead